android/assets/jsons/translations/Indonesian.properties

```diff
@@ -5620,7 +5620,7 @@ Desdemona = Desdemona
 Anthea = Anthea
 Aeneas = Aeneas
 Leander = Leander
-Christianity = Kristen
+Christianity = Kekristenan
```

Comment:
For the religions like Christianity, Hinduism, and Buddhism, even though the updated translations are correct in a literal way, here I think it is better to display the religion like what is shown in your identity card like Kristen, Hindu, or Buddha (btw the correct Buddhism uses double d)